apps/passport-client/new-components/shared/Input.tsx

```diff
@@ -44,17 +45,22 @@ export const BigInput2 = styled.input<{
   error?: string;
   hideArrows?: boolean;
   variant: "primary" | "secondary";
+  hasRightIcons?: boolean;
 }>`
   width: 100%;
   height: 54px;
   border-radius: 200px;
   padding: 8px 24px;
+  padding-right: ${({ hasRightIcons }) => (hasRightIcons ? "44px" : "24px")};
```

Comment:
lets insead pass an `rightIconSize` and use it as the flag and the amount of padding to add

Reply:
also fix lints